src/css/_icons.scss

```diff
@@ -11,7 +11,7 @@
   background-repeat: no-repeat;
   background-position: center;
 
-  fill: #FFFFFF;
+  fill: currentColor;
```

Comment:
I could be wrong here and I haven't tested, but should this be [currentcolor](https://www.w3schools.com/colors/colors_currentcolor.asp)?

Reply:
Looks like it https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword
though, I think it's case-insensitive

Reply:
@wseymour15, @gkatsev thanks for the review! 

Would you like me to adjust the value ? If it helps to make a decision, here's what the W3C says https://www.w3.org/TR/css-color-3/#currentcolor

Reply:
It's always fun when docs conflict 😄 You can leave as is. No strong feelings after reading that.

Reply:
yeah, interesting that mdn doesn't document it like the spec. I would always err on the side of the spec, if possible.